build-tests.cmd

```diff
@@ -0,0 +1,24 @@
+@echo off
```

Comment:
Make this toggleable like we now do in build.cmd

@if "%_echo%" neq "on" echo off